NuRadioReco/detector/detector.py

```diff
@@ -403,6 +403,8 @@ def get_absolute_position(self, station_id):
             easting = res['pos_easting'] * unit_xy
         if(res['pos_northing'] is not None):
             northing = res['pos_northing'] * unit_xy
+        if(res['pos_altitude'] is not None):
+            altitude = res['pos_altitude'] * unit_xy
```

Comment:
this wasn't a bug, please remove this change. The altitude is always saved in the default unit which is meters. The only exception is the special coordinate system at the South Pole where x/y is specified in feet.

Reply:
And look at the docstring. The altitude is relative to the snow level at the time of the measurement. Which is a good thing for neutrino detectors,... but I can see that this is not ideal for cosmic rays. Not sure how to solve it... 
Maybe define two functions `get_absolute_position_wrt_surface` `get_absolute_position_absolute` hmm, but the naming is odd. `absolute_position` should be absolute. So we should rather rename the current function to `get_position_wrt_surface`

Reply:
Ah sorry I did not catch that. Yes indeed we would need a function which returns the absolute position. Should I add this and rename the above function to `get_position_wrt_surface`?

Reply:
Hi,
how do you think about this? two getter functions? I did not really understand why the function does not return any other altitude other than 0. You don't have snow accumulation? Just out of curiosity :)